File: leetcode_py/ReverseInteger.py
def reverse(x):
	if (x>0):
		x=str(x)
		y = int(x[-1::-1])
		return y
	elif (x==0): return x
	else:
		x=str(x)[1:]
		y = int(x[-1::-1])
		return -y

print reverse(100000000)

File: leetcode_java/ReverseInteger.java
package leetcode_140;

public class ReverseInteger {
	public int reverse(int x) {
		// IMPORTANT: Please reset any member data you declared, as
		// the same Solution instance will be reused for each test case.
		if(x>0){
			String y = Integer.toString(x);
			String reverse = new StringBuffer(y).reverse().toString();
			return Integer.parseInt(reverse);
		}
		if (x<0){
			x = -x;
			String y = Integer.toString(x);
			String reverse = new StringBuffer(y).reverse().toString();
			return -Integer.parseInt(reverse);
		}
	    return 0;
	}
}


























